en_us.lang

```diff
@@ -157,200 +157,200 @@ gui.xaero_transfer_all=Transfer All Waypoints
-gui.xaero_box_compass_over_wp=Render compass letters (N, E, S, W) over the on-map waypoints.,
-gui.xaero_toggle_map_waypoints=Toggle On-map Waypoints
-gui.xaero_entity_nametags=Entity Name Tags
+gui.xaero_box_distance=Specifies, for which in-game waypoints to show the distance to.
+gui.xaero_box_always_distance=Display distance even when the waypoint is very close. This setting does not override the "Distance to WP" option.
+gui.xaero_box_arrow_scale=Scale of the arrow used in the non-rotating variant of the minimap and several other cases.
+gui.xaero_box_arrow_color=Color of the arrow used in the non-rotating variant of the minimap and several other cases.
+gui.xaero_box_lock_player_heads=Display player heads instead of colored dots even if the Tab key is not pressed.
+gui.xaero_box_cave_zoom=How much to zoom further in the minimap when the cave mode is active.
+gui.xaero_box_entity_depth=Display entities darker on the minimap depending on their Y level relative to you.
+gui.xaero_box_height_limit=Maximum Y-level distance for entities until they are no longer displayed on the minimap.
+gui.xaero_box_compass_over_wp=Render compass letters (N, E, S, W) over the on-map waypoints.
+gui.xaero_toggle_map_waypoints=Toggle On-Map Waypoints
+gui.xaero_entity_nametags=Entities With Name Tags
 gui.xaero_minimap_text_align=Info Text Alignment
 gui.xaero_center=Center
 gui.xaero_left=Left
 gui.xaero_right=Right
-gui.xaero_show_angles=Show View Angles
-gui.xaero_box_entity_nametags=Display entity name under the dot for entities that have a name tag.
-gui.xaero_box_teleport_default_command=The command used for waypoints teleportation if a server-specific command is not set in the waypoints menu Options screen.
-gui.xaero_waypoints_edit_mode_individually=One-by-one
-gui.xaero_waypoints_edit_mode_all=All
+gui.xaero_show_angles=Display Facing Angles
+gui.xaero_box_entity_nametags=Display entity name under the dot on the minimap for entities that have name tags.
+gui.xaero_box_teleport_default_command=The command used for waypoint teleportation if a world-specific or a server-specific command is not set in the Options menu on the waypoint screen.
+gui.xaero_waypoints_edit_mode_individually=One by One
+gui.xaero_waypoints_edit_mode_all=All at Once
 gui.xaero_compass=Compass
-gui.xaero_cave_maps_depth=Cave Maps Depth
+gui.xaero_cave_maps_depth=Cave Mode Depth
 gui.xaero_waypoints_edit_reset=Reset
 gui.xaero_main_entity_as=Main Entity As
 gui.xaero_arrow=Arrow
 gui.xaero_dot=Dot
 gui.xaero_crosshair=Crosshair
-gui.xaero_teleport_coordinates_hidden=Unable to teleport because it would tell you the waypoint coordinates. Disable "Hide Waypoint Coords" to be able to freely teleport again.
-gui.xaero_hide_wp_coords=Hide Waypoint Coords
+gui.xaero_teleport_coordinates_hidden=Unable to teleport, because it could expose the waypoint coordinates. Disable the "Hide Waypoint Coordinates" option to be able to freely teleport again.
```

Comment:
I used "would" here to make it scarier, as it should be, so I'm not sure if we should change it to "could".

Reply:
I'm not a linguist, so perhaps you shouldn't listen to me, but to me "something bad would happen" and "something bad could happen" are nearly identical in terms of fear. "Something bad **will** happen" is scarier but the difference is rather small, something like between "may" and "might".

Reply:
IDK, "would" sounds like a much higher probability than "could" to me. Isn't it just "will" in the past tense?

Reply:
Yes, it is, but it doesn't matter here, as we are talking about the future. If you definitely think that "would" is the best one here (instead of "could" or "will"), let me know.

Reply:
I do think it's better but do you disagree?

Reply:
Yes, I do, but previously I said "the difference is rather small", so I'm going to use "would" anyway.